client/src/main/java/org/evosuite/instrumentation/coverage/DefUseInstrumentation.java

```diff
@@ -18,7 +18,7 @@
  * License along with EvoSuite. If not, see <http://www.gnu.org/licenses/>.
```

Comment:
Please, do not change the indentation of lines that you have not modified.  The pull request is already super long and these extra and unnecessary changes only make it even harder to review.